pkg/dflog/log.go

```diff
@@ -72,7 +77,7 @@ func getLumberjack(l *logrus.Logger) *lumberjack.Logger {
 func WithLogFile(f string, maxSize, maxBackups int) Option {
 	return func(l *logrus.Logger) error {
 		if f == "" {
-			return nil
+			f = filepath.Join("logs", "dfdaemon.log")
```

Comment:
In fact, not only dfdaemon will use this function, so do with `dfget` and `supenrode`.

Reply:
Ok, I rollbacked the change of this line, and set the default log path for dfget/dfdaemon in caller (See my new commits).

As for supernode, I didn't touch its code. Because it [writes to two different log files](https://github.com/dragonflyoss/Dragonfly/blob/def2b217d4faab07dd9fa03e527d3dc3cc106b8a/cmd/supernode/app/root.go#L84-L92), a single `logConfig.path` property is not enough to configure the location of these files.

For now, this PR configures the log path of dfget/dfdaemon, not for supernode.